### DISCUSSION TITLE
Reinstalling Node

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#6095] [General]
Hi,

I'm reinstalling the Mesh and I'm trying to install Node with _nodesource install_ but i'm getting the error : nodesource command not found.
It's not about the Mesh itself but any help would be appreciated.

**Comment by si458** [comment]
check the official nodejs site, they now recommend to use NVM instead so u can easily switch between nodejs versions
https://nodejs.org/en/download/package-manager
make sure u select node 20 lts, linux and nvm
